src/js/angular/rest/repositories.rest.service.js

```diff
@@ -141,4 +142,28 @@ function RepositoriesRestService($http) {
             }
         });
     }
+
+    function downloadResultsAsFile(apiUrl, queryParams, acceptHeader) {
```

Comment:
This is strange. Why not providing only the repository id and avoid hardcoding the endpoint in the controller?

Reply:
Yes, no reason for this I just extracted the function. I will change the argument